docs/community.aws.aws_acm_module.rst

```diff
@@ -42,8 +42,8 @@ Requirements
 The below requirements are needed on the host that executes this module.
 
 - python >= 3.6
-- boto3 >= 1.15.0
-- botocore >= 1.18.0
+- boto3 >= 1.16.0
+- botocore >= 1.19.0
```

Comment:
Is this correct based on the min version bump being done only on v3 ?
https://github.com/ansible-collections/community.aws/pull/809/files

Reply:
Good question. I think the patchback config (_and the stable-2_) branch was created later. https://github.com/ansible-collections/community.aws/pull/812

So the branch included accidentally this change.

cc @tremble

Reply:
No, that should *not* be backported.